### DISCUSSION TITLE
forrtl: severe (157): Program Exception - access violation

### DISCUSSION THREAD

**Discussion by ESH-y** [firemodels/fds#12919] [Q&A]
HI

Has anyone else run into this error by chance? This is the first time I have come across it.
 I would appreciate some help in guiding me to solve the error.

forrtl: severe (157): Program Exception - access violation
Image              PC                Routine            Line        Source
fds.exe            00007FF636102BF6  Unknown               Unknown  Unknown
fds.exe            00007FF6360D6D1E  Unknown               Unknown  Unknown
fds.exe            00007FF6360A19FE  Unknown               Unknown  Unknown
fds.exe            00007FF636A23370  Unknown               Unknown  Unknown
KERNEL32.DLL       00007FFFC02A7974  Unknown               Unknown  Unknown
ntdll.dll          00007FFFC30BA2F1  Unknown               Unknown  Unknown

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 0 PID 14820 RUNNING AT NO24
=   EXIT STATUS: -1 (ffffffff)
===================================================================================

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 1 PID 17504 RUNNING AT NO24
=   EXIT STATUS: -1 (ffffffff)
===================================================================================

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 2 PID 15532 RUNNING AT NO24
=   EXIT STATUS: -1 (ffffffff)
===================================================================================

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 4 PID 9968 RUNNING AT NO24
=   EXIT STATUS: -1 (ffffffff)
===================================================================================

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 5 PID 14700 RUNNING AT NO24
=   EXIT STATUS: -1 (ffffffff)
===================================================================================

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 6 PID 16244 RUNNING AT NO24
=   EXIT STATUS: -1 (ffffffff)
===================================================================================

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 7 PID 14212 RUNNING AT NO24
=   EXIT STATUS: -1 (ffffffff)
===================================================================================

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 8 PID 15428 RUNNING AT NO24
=   EXIT STATUS: -1 (ffffffff)
===================================================================================

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 9 PID 11688 RUNNING AT NO24
=   EXIT STATUS: -1 (ffffffff)
===================================================================================

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 10 PID 17196 RUNNING AT NO24
=   EXIT STATUS: -1 (ffffffff)
===================================================================================

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   RANK 11 PID 4728 RUNNING AT NO24
=   EXIT STATUS: -1 (ffffffff)
===================================================================================

**Comment by mcgratta** [comment]
This is a common error statement telling us that something went wrong in the code, like an array overshoot or similar numerical error. Post your input file and tell us what your operating system is, version of FDS, and when the error occurs.

**Comment by ncwu-chc** [comment]
I also encountered this issue. After testing, I thinkit is caused by the mixing of materials specified in the model. You can try running with only one material set, and then gradually add other materials, ensuring that the various materials are not excessively mixed together in a chaotic manner.
